src/manifest.json

```diff
@@ -1,15 +1,21 @@
 {
-  "name": "SIA",
-  "theme_color": "#b1624d",
-  "background_color": "#fafafa",
-  "display": "standalone",
-  "Scope": "/",
-  "start_url": "/",
+  "background_color": "$SIGNALS_BACKGROUND_COLOR",
+  "display": "fullscreen",
   "icons": [
     {
-      "src": "favicon.png",
+      "src": "$SIGNALS_IOS_ICON",
+      "sizes": "180x180",
+      "type": "image/png"
+    },
+    {
+      "src": "$SIGNALS_ANDROID_ICON",
+      "sizes": "192x192",
```

Comment:
The icon seems skewed. I would suggest to get the icons from https://mijn.amsterdam.nl/favicons/android-chrome-192x192.png and https://mijn.amsterdam.nl/favicons/android-chrome-256x256.png; those have the correct dimensions.